test/compilable/ddoc4899.d

```diff
@@ -4,8 +4,6 @@
 /*
 TEST_OUTPUT:
 ---
-compilable/ddoc4899.d(16): Warning: Ddoc: Stray '('. This may cause incorrect Ddoc output. Use $(LPAREN) instead for unpaired left parentheses.
-compilable/ddoc4899.d(16): Warning: Ddoc: Stray ')'. This may cause incorrect Ddoc output. Use $(RPAREN) instead for unpaired right parentheses.
```

Comment:
Same problem as https://github.com/dlang/dmd/pull/7043#discussion_r130244754.

I'm surprised this change isn't causing a merge conflict!